tasks/devcontainer.py

```diff
@@ -92,11 +95,18 @@ def setup(
         "--name",
         "datadog_agent_devcontainer",
     ]
+    if sys.platform != "win32":
+        # Save the current user's UID and GID to a local `.env` file
+        env_file = os.path.join(DEVCONTAINER_DIR, ".env")
+        with open(env_file, "w") as f:
+            f.write(f"UID={os.getuid()}\n")
+            f.write(f"GID={os.getgid()}\n")
+        devcontainer["runArgs"].append(f"--env-file={env_file}")
+
     devcontainer["features"] = {}
-    devcontainer["remoteUser"] = "datadog"
     devcontainer["mounts"] = [
         "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind,consistency=cached",
-        "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached",
+        "source=${localEnv:HOME}${localEnv:USERPROFILE}/.ssh,target=/home/vscode/.ssh,type=bind,consistency=cached",
```

Comment:
This fallback example is also in the documentation https://containers.dev/implementors/json_reference/#variables-in-devcontainerjson